app/assets/javascripts/misc/pw-strength.js

```diff
@@ -46,6 +54,9 @@ function analyzePw() {
   const pwCntnr = document.getElementById('pw-strength-cntnr');
   const pwStrength = document.getElementById('pw-strength-txt');
   const pwFeedback = document.getElementById('pw-strength-feedback');
+  const submit = document.querySelector('input[type="submit"]');
+
+  disableSubmit(submit, 0);
```

Comment:
Probably just semantics, but is there an advantage to this over just doing `submit.setAttribute('disabled', true);`?